main.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import "fmt"
+
+type Man struct {
+	Name     string
+	LastName string
+	Age      int
+	Gender   string
+	Crimes   int
+}
+
+func main() {
+	man1 := Man{"Alexander", "Pichushkin", 49, "Male", 49}
+	man2 := Man{"Mikhail", "Popkov", 59, "Male", 78}
+	man3 := Man{"Andrei", "Chikatilo", 57, "Male", 52}
+	man4 := Man{"Javed", "Iqbal", 40, "Male", 100}
+	man5 := Man{"Luis", "Garavito", 66, "Male", 143}
+
+	var people = map[string]Man{
+		"Alexander": man1,
+		"Mikhail":   man2,
+		"Andrei":    man3,
+		"Javed":     man4,
+		"Luis":      man5,
+	}
```

Comment:
можно упростить до следующего варианта, так код выглядит нагляднее  
```suggestion
	var people = map[string]Man{
		"Alexander": Man{"Alexander", "Pichushkin", 49, "Male", 49},
		"Mikhail":   Man{"Mikhail", "Popkov", 59, "Male", 78},
		"Andrei":    Man{"Andrei", "Chikatilo", 57, "Male", 52},
		"Javed":     Man{"Javed", "Iqbal", 40, "Male", 100},
		"Luis":      Man{"Luis", "Garavito", 66, "Male", 143},
	}
```